launcher/util.go

```diff
@@ -3,11 +3,22 @@ package launcher
 import (
 	"context"
 	"fmt"
+	"io"
 
+	"github.com/google/go-tpm/legacy/tpm2"
 	"google.golang.org/api/impersonate"
 	"google.golang.org/api/option"
 )
 
+// TPMDAParams holds TPM Dictionary Attack parameters
+type TPMDAParams struct {
+	LockoutCounter      uint32
+	MaxAuthFail         uint32 // maxTries
+	LockoutInterval     uint32 // recoveryTime
+	LockoutRecovery     uint32 // lockoutRecovery
```

Comment:
Please add a note explaining the mapping discrepancy between capabilities and the command